about/about-kubernetes-networking.md

```diff
@@ -125,10 +125,10 @@ as IPv4 or IPv6 addresses.
 - {% include open-new-window.html text='Video: Everything you need to know about Kubernetes pod networking on AWS'
   url='https://www.projectcalico.org/everything-you-need-to-know-about-kubernetes-pod-networking-on-aws/' %}
 - {% include open-new-window.html text='Video: Everything you need to know about Kubernetes pod networking on Azure'
-  url='https://www.projectcalico.org/everything-you-need-to-know-about-kubernetes-pod-networking-on-azure/' %}
+  url='https://www.projectcalico.org/everything-you-need-to-know-about-kubernetes-networking-on-azure/' %}
```

Comment:
I had to do some judgement calls on links like this one since we are redirecting people outside of documentation I thought these should open in new window.

@lxpollitt

Reply:
Yep, this looks right now